test/ctor-eval/results.wast.out

```diff
@@ -0,0 +1,7 @@
+(module
```

Comment:
Are these tests a good candidate for converting to `lit`?    Should we do that before add all these new ones?  @tlively ?

Reply:
Hmm, these might work as lit tests, yeah.

Do we want to do that now? I'm not sure of the overall priority of that migration atm.

Would we port these tests using a script anyhow like other groups? If so then I think there's no harm to adding more now. But if it would be manual then perhaps new tests should be lit...

Reply:
No worries.. I don't want to block this.. I just recently was reminded how annoying it is to try to run one of these tests in isolation so I was hoping we could keep the momentum going, but it doesn't have to happen as part of this change.

Reply:
BTW how do you run individual tests like this?  Or do you just run the whole suite?

Reply:
For an individual test I just write out the command, something like `wasm-ctor-eval test/ctor-eval/NAME.wat --ctors test1 -o /dev/stdout -S`.

Is there a way to run individual lit tests?

Reply:
Yes, you just name the file on the command line.  Each lit test is by-definition described by a single file so it fairly obvious and you can use tab completion and wildcards.   This is probably my number 1 reason for wanting to make the switch:  For example, you see a test failure on the bot (it has a name which is the name of the file) and you can simply run that one test locally to reproduce.   With the current binaryen test suite (the non-lit parts) its really hard to see what exactly failed, and how exactly one might re-run the failing part (IMHO).

For example, when I work on lld I'm always running commands like this:

```
$ ../llvm-build/bin/llvm-lit lld/test/wasm/
$ ../llvm-build/bin/llvm-lit lld/test/wasm/*pic*
```

binaryen-lit works exactly the same way I believe.

Reply:
Anyway, this change lgtm :)